guide/README.md

```diff
@@ -455,6 +455,56 @@ used to bump the `Mi 4` note up by one octave and then to bump the
 | | | For example: Transposing C-D-E-F by 4 (fifth) will give us G-A-B-C
 | | | To transpose an octave: shift by the mode length (7 in major scales) up or down. |
 
+#### <a name="Set Key"></a>3.2.5 Set Key
+
+![alt tag](./setkey1.svg "scalar transposition")
+
+
+The Set key block is used to set the key and mode, eg C Major. 
+It allows you to change both the mode and key of how solfege is mapped to the notes.
```

Comment:
It is not just solfege. More important is scalar step.

It is used to change both the mode and key of the current scale. (The current scale is used to define the mapping of Solfege to notes and also the number of half steps take by the the *Scalar step* block.